pkg/metrics/backup.go

```diff
@@ -186,3 +192,25 @@ func (rm RestoreM) UpdateBatchSize(clusterID uuid.UUID, host string, size int64)
 
 	rm.batchSize.With(l).Add(float64(size))
 }
+
+// SetRemainingBytes sets restore "remaining_bytes" metric.
+func (rm RestoreM) SetRemainingBytes(clusterID uuid.UUID, location backupspec.Location, snapshotTag string, keyspaces []string, remainingBytes int64) {
+	l := prometheus.Labels{
+		"cluster":      clusterID.String(),
+		"location":     location.String(),
+		"snapshot_tag": snapshotTag,
+		"keyspaces":    strings.Join(keyspaces, ","),
```

Comment:
Do we want to change labels on metrics?
Why `manifest_path` has been replaced with `location`? 
Is `snapshot_tag` really needed here?
Why slice of keyspaces and not keyspace + table?